components/apimgt/org.wso2.carbon.apimgt.persistence/src/main/java/org/wso2/carbon/apimgt/persistence/RegistryPersistenceImpl.java

```diff
@@ -4158,8 +4175,7 @@ public void changeApiProvider(String providerName, String apiId, String org)
                 if (log.isDebugEnabled()) {
                     log.debug("Changing the provider name of API with id: " + apiId + " to " + providerName);
                 }
-                artifact.setAttribute(APIConstants.API_OVERVIEW_PROVIDER, RegistryPersistenceUtil
```

Comment:
Does this change the current behaviour for email as username users?

Reply:
It will not break because we have reverted the changes introduced in https://github.com/wso2-support/carbon-apimgt/pull/7450. The new code changes no longer copy artifacts to the newly appointed API provider. Instead, the original user's path will remain the unique path for the API, just as we store the API information under the original user's registry path.